sympy/plotting/plot.py

```diff
@@ -289,6 +289,121 @@ def extend(self, arg):
             raise TypeError('Expecting Plot or sequence of BaseSeries')
 
 
+class PlotGrid(object):
+    """This class helps to plot subplots from already created sympy plots
+    in a single figure.
+
+    Examples
+    ========
+
+    >>> from sympy import symbols
+    >>> from sympy.plotting.plot import plot, plot3d, PlotGrid
+    >>> x, y = symbols('x, y')
+    >>> p1 = plot(x, x**2, x**3, (x, -5, 5))
+    >>> p2 = plot((x**2, (x, -6, 6)), (x, (x, -5, 5)))
+    >>> p3 = plot(x**3, (x, -5, 5))
+    >>> p4 = plot3d(x*y, (x, -5, 5), (y, -5, 5))
+
+    Plotting vertically in a single line
+
+    >>> PlotGrid(2, 1 , p1, p2)
+    PlotGrid object containing:
+    Plot[0]:Plot object containing:
+    [0]: cartesian line: x for x over (-5.0, 5.0)
+    [1]: cartesian line: x**2 for x over (-5.0, 5.0)
+    [2]: cartesian line: x**3 for x over (-5.0, 5.0)
+    Plot[1]:Plot object containing:
+    [0]: cartesian line: x**2 for x over (-6.0, 6.0)
+    [1]: cartesian line: x for x over (-5.0, 5.0)
+
+    Plotting horizontally in a single line
+
+    >>> PlotGrid(1, 3 , p2, p3, p4)
+    PlotGrid object containing:
+    Plot[0]:Plot object containing:
+    [0]: cartesian line: x**2 for x over (-6.0, 6.0)
+    [1]: cartesian line: x for x over (-5.0, 5.0)
+    Plot[1]:Plot object containing:
+    [0]: cartesian line: x**3 for x over (-5.0, 5.0)
+    Plot[2]:Plot object containing:
+    [0]: cartesian surface: x*y for x over (-5.0, 5.0) and y over (-5.0, 5.0)
+
+    Plotting in a grid form
+
+    >>> PlotGrid(2, 2, p1, p2 ,p3, p4)
+    PlotGrid object containing:
+    Plot[0]:Plot object containing:
+    [0]: cartesian line: x for x over (-5.0, 5.0)
+    [1]: cartesian line: x**2 for x over (-5.0, 5.0)
+    [2]: cartesian line: x**3 for x over (-5.0, 5.0)
+    Plot[1]:Plot object containing:
+    [0]: cartesian line: x**2 for x over (-6.0, 6.0)
+    [1]: cartesian line: x for x over (-5.0, 5.0)
+    Plot[2]:Plot object containing:
+    [0]: cartesian line: x**3 for x over (-5.0, 5.0)
+    Plot[3]:Plot object containing:
+    [0]: cartesian surface: x*y for x over (-5.0, 5.0) and y over (-5.0, 5.0)
+
+    """
```

Comment:
Please make these display plots as in: https://docs.sympy.org/dev/modules/plotting.html

Reply:
Should it be like this?
```
>>> PlotGrid(1, 3 , p2, p3, p4)
PlotGrid object containing:
Plot[0]:
Plot object containing:
[0]: cartesian line: x**2 for x over (-6.0, 6.0)
[1]: cartesian line: x for x over (-5.0, 5.0)
Plot[1]:
Plot object containing:
[0]: cartesian line: x**3 for x over (-5.0, 5.0)
Plot[2]:
Plot object containing:
[0]: cartesian surface: x*y for x over (-5.0, 5.0) and y over (-5.0, 5.0)
```

Reply:
I don't know, check the source of the other files on how to do it.

Reply:
I have made the docs display the plots. It now looks like this:

![image](https://user-images.githubusercontent.com/29159034/55670216-6475d900-589f-11e9-9d9a-a079fe4c2cc7.png)